packages/tables/src/Columns/TextInputColumn.php

```diff
@@ -2,22 +2,29 @@
 
 namespace Filament\Tables\Columns;
 
+use BackedEnum;
 use Closure;
+use Filament\Actions\Action;
+use Filament\Forms\Components\Concerns\HasAffixes;
 use Filament\Forms\Components\Concerns\HasExtraInputAttributes;
 use Filament\Forms\Components\Concerns\HasInputMode;
 use Filament\Forms\Components\Concerns\HasStep;
 use Filament\Support\Components\Contracts\HasEmbeddedView;
 use Filament\Support\Enums\Alignment;
 use Filament\Support\Facades\FilamentAsset;
 use Filament\Support\RawJs;
+use Filament\Support\View\Components\InputComponent\WrapperComponent\IconComponent;
 use Filament\Tables\Columns\Contracts\Editable;
 use Filament\Tables\Table;
+use Illuminate\Contracts\Support\Htmlable;
 use Illuminate\Support\Js;
+use Illuminate\View\ComponentAttributeBag;
 
 class TextInputColumn extends Column implements Editable, HasEmbeddedView
 {
     use Concerns\CanBeValidated;
     use Concerns\CanUpdateState;
+    use HasAffixes;
```

Comment:
Add any other methods from this trait so that it can be removed, we shouldn't reference forms traits from tables if we can